contributors/devel/sig-testing/writing-good-e2e-tests.md

```diff
@@ -265,7 +265,7 @@ operation compared to running the same test locally. On the other hand, a too
 long timeout can be annoying when trying to debug tests locally.
 
 The framework provides some [common
-timeouts](https://github.com/kubernetes/kubernetes/blob/eba98af1d8b19b120e39f3/test/e2e/framework/timeouts.go#L44-L109)
+timeouts](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/timeouts.go#L44-L109)
```

Comment:
Let's use the same stable commit as below
```suggestion
timeouts](https://github.com/kubernetes/kubernetes/blob/1e84987baccbccf929eba98af1d8b19b120e39f3/test/e2e/framework/timeouts.go#L44-L109)
```

Reply:
Thanks for the review. I will apply the suggestion.